docs/current_docs/adopting.mdx

```diff
@@ -0,0 +1,16 @@
+---
+slug: /adopting
+---
+
+# Adopting Dagger
+
+You’ve done the [quickstart](quickstart/index.mdx), and now you’re probably asking yourself, "What should I do next?" 
+
+We get it, looking at that many lines of YAML can be intimidating. 
+Since each pipeline is unique in tooling and functionally, our general answer to is usually "start simple". Pick a part of your pipeline to Daggerize first, and then continue to daggerize from there. 
```

Comment:
Two nits, otherwise LGTM 

```diff
- our general answer to is
+ our general answer is to
```

`Daggerize` or `daggerize`?

Reply:
🙏 fixed word order, removed "usually", standardized on "Daggerize" as it's most common in our docs.

Reply:
To pick up on `Daggerize`, `Daggerizes`, `Daggerized`, `Daggerizing`, etc while ignoring binary files (`-I`) and counting occurrences (`-o`) not lines, as they might have multiple occurrences. Recursive through subdirs (`-r`) and with line numbers (`-n`). Excluding `.json`, `.js`, and `.ts` files.
```
docs ➤ grep -ronI Daggeriz | grep -v json: | grep -v js: | grep -v ts:
./current_docs/index.mdx:25:Daggeriz
./current_docs/quickstart/conclusion.mdx:12:Daggeriz
./current_docs/quickstart/conclusion.mdx:15:Daggeriz
./current_docs/quickstart/conclusion.mdx:15:Daggeriz
./current_docs/quickstart/conclusion.mdx:15:Daggeriz
./current_docs/quickstart/conclusion.mdx:18:Daggeriz
./current_docs/quickstart/conclusion.mdx:24:Daggeriz
./current_docs/quickstart/daggerize.mdx:3:Daggeriz
./current_docs/quickstart/daggerize.mdx:11:Daggeriz
./current_docs/quickstart/daggerize.mdx:13:Daggeriz
./current_docs/manuals/user/visualization/cloud-get-started.mdx:12:Daggeriz
./current_docs/adopting.mdx:10:Daggeriz
./current_docs/adopting.mdx:10:Daggeriz
./current_docs/adopting.mdx:12:Daggeriz
```

```
docs ➤ grep -ronI daggeriz | grep -v json: | grep -v js: | grep -v ts:
./current_docs/quickstart/daggerize.mdx:2:daggeriz
./current_docs/quickstart/daggerize.mdx:74:daggeriz
./current_docs/quickstart/daggerize.mdx:89:daggeriz
./current_docs/quickstart/daggerize.mdx:104:daggeriz
```

🇬🇧 check - looking for anything, ignoring case (`-i`).
docs ➤ grep -ronIi daggeris | grep -v json: | grep -v js: | grep -v ts:
∅